### PR TITLE
Feature/cv2 4630 remove article from item

### PR DESCRIPTION
**Pull request by sarahkeh**
## Description

Clicking X will open a confirmation dialog that a user can confirm the removal of an article from a media item.  List will update and no longer show the article.
 
CV2-4630

## Type of change

- [ ] Performance improvement and/or refactoring (non-breaking change that keeps existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Security mitigation or enhancement
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Automated test (add or update automated tests)

## How has this been tested?

Added and removed ExplainerItems

## Things to pay attention to during code review

Ensure list updates

## Checklist

- [X] I have performed a self-review of my own code
- [X] I've made sure my branch is runnable and given good testing steps in the PR description
- [X] I considered secure coding practices when writing this code. Any security concerns are noted above.
- [X] I have commented my code in hard-to-understand areas, if any
- [ ] I have made needed changes to the README
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] If I implemented any new components, they are self-contained, their `propTypes` are declared and they use React Hooks and, if data-fetching is required, they use Relay Modern with fragment containers
- [ ] If my components involve user interaction - specifically button, text fields, or other inputs - I have added a [BEM-like class name](https://meedan.atlassian.net/wiki/spaces/ENG/pages/1327628289/Naming+conventions+for+interactive+elements) to the element that is interacted with
- [ ] To the best of my knowledge, any new styles are applied according to the design system
- [ ] If I added a new external dependency, I included a rationale for doing so and an estimate of the change in bundle size (e.g., checked in https://bundlephobia.com/)

### CODE REVIEW

**Comment by amoedoamorim** [comment]
@sarahkeh please create a check-api branch with the same name as this one and re-run the Travis job

File: src/app/components/article/MediaArticleCard.js [hunk 1]
```diff
@@ -117,6 +124,7 @@ MediaArticleCard.propTypes = {
   publishedAt: PropTypes.number, // Timestamp
   variant: PropTypes.oneOf(['explainer', 'fact-check']),
   onClick: PropTypes.func,
+  id: PropTypes.number.isRequired,
```
Comment (amoedoamorim) [inline src/app/components/article/MediaArticleCard.js:127]: Question: is this really a number or is it a string?

Comment (sarahkeh) [reply src/app/components/article/MediaArticleCard.js]: Oh shoot, you're right, fixed

**Review by amoedoamorim — APPROVED** [review]
:lgtm:

File: src/app/components/article/MediaArticleCard.js (hunk 1 above)
Comment (amoedoamorim) [reply src/app/components/article/MediaArticleCard.js]: Awesome, thanks!

**Pull request merged**